lang/ilias_bg.lang

```diff
@@ -12183,7 +12183,6 @@ buddysystem#:#buddy_request_ignored#:#You ignored the user.###30 08 2015 new var
 common#:#no_gallery_users_available#:#There are currently no users in the gallery.###30 08 2015 new variable
 crs#:#crs_mem_contact#:#Support Contact###30 08 2015 new variable
 awrn#:#awareness_now_online#:#Online Now###30 08 2015 new variable
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]###30 08 2015 new variable
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.